homeassistant/components/config/customize.py

```diff
@@ -0,0 +1,45 @@
+"""Provide configuration end points for Customize."""
+import asyncio
+
+from homeassistant.components.config import EditKeyBasedConfigView
+from homeassistant.components import SERVICE_RELOAD_CORE_CONFIG
+from homeassistant.config import DATA_CUSTOMIZE
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.core import DOMAIN as CORE_DOMAIN
+
+CONFIG_PATH = 'customize.yaml'
+
+
+@asyncio.coroutine
+def async_setup(hass):
+    """Set up the Customize config API."""
+    @asyncio.coroutine
+    def hook(hass):
+        """Hook to call after updating cutomization file."""
+        yield from hass.services.async_call(
+            CORE_DOMAIN, SERVICE_RELOAD_CORE_CONFIG)
+
+    class CustomizeConfigView(EditKeyBasedConfigView):
+        """Configure a list of entries."""
+
+        def _get_value(self, data, config_key):
```

Comment:
We should update the method signatures for get and write value to include `hass`, that way you don't have to do declare the class inline.